<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managementDetail</class>
 <widget class="QWidget" name="managementDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="proinfor_lb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品描述：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pronumber_line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>510</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改商品</string>
   </property>
  </widget>
  <widget class="QLabel" name="upFail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改失败！</string>
   </property>
  </widget>
  <widget class="QLabel" name="pronumber_lb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品数量：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="proinfor_text">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="proprice_line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
  </widget>
  <widget class="QLabel" name="upSuccess">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改成功！</string>
   </property>
  </widget>
  <widget class="QLabel" name="proprice_lb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品价格：￥</string>
   </property>
  </widget>
  <widget class="QLabel" name="proname_lb">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="proname_line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prodiscount_line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);</string>
   </property>
   <property name="text">
    <string>商品折扣：</string>
   </property>
  </widget>
  <widget class="QLabel" name="prodiscount_lb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品折扣：</string>
   </property>
  </widget>
  <widget class="QLabel" name="prodiscount_lb2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
